lib/gpio/pin.toit

```diff
@@ -391,7 +393,10 @@ gpio-init_:
   #primitive.gpio.init
 
 gpio-use_ resource-group num allow-restricted:
-  #primitive.gpio.use
+  #primitive.gpio.use:
+    if it == RESTRICTED-PIN-ERROR_:
```

Comment:
What is going on with this "RESTRICTED" <-> "RESTRICTED_PIN" dance?

Reply:
removed.
done.